### PR TITLE
Cherry-pick: Rename isMasterNode and Fix network utils

### PR DESCRIPTION
**Pull request by lzhecheng**
#### What this PR does / why we need it:
It cherry-picks a commit to Rename isMasterNode and fix network utils

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by lzhecheng** [comment]
This commit may fix the "Found no or more than 1 virtual network in resource group same as cluster name' error in release-0.7 branch.

File: pkg/consts/consts.go [hunk 1]
```diff
@@ -83,6 +83,14 @@ const (
 
 	// NodeLabelRole specifies the role of a node
 	NodeLabelRole = "kubernetes.io/role"
+	// MasterNodeRoleLabel specifies is the master node label for a node
+<<<<<<< HEAD
```
Comment (andyzhangx) [inline pkg/consts/consts.go:87]: resolve merge conflict

Comment (lzhecheng) [reply pkg/consts/consts.go]: Thanks for catching this. Updated.

**Comment by lzhecheng** [comment]
I didn't realize these 2 commits are already included in this cherry-pick PR: #982.
Closing this one.

**Pull request closed without merging**